CIP-0010/registry.schema.json

```diff
@@ -1,64 +1,25 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0010",
-    "type": "array",
```

Comment:
why are we removing things from the schema?

Reply:
could removing this break anyone existing implmentations?

Reply:
Impossible to know I think? I don't think we have access to statistics on the usage of this file?
But as mentioned, there is no need to do this change, the main reason for this is to add the validation step to prevent breaking changes.

Reply:
@Ryun1 @Godspeed-exe we are confirming now in CIP meeting that @Crypto2099 has assessed the consequences to existing implementations and ruled them out (my words, not his).

Reply:
Understood thanks @Godspeed-exe